config/_default/menus/menus.en.yaml

```
@@ -4523,6 +4523,11 @@ main:
     url: synthetics/browser_tests/advanced_options
     parent: synthetics_browser_tests
     weight: 303
+  - name: Authentication in Browser Tests 
+    identifier: synthetics_browser_tests_authentication
+    url: synthetics/browser_tests/app-that-requires-login
+    parent: synthetics_browser_tests
+    weight: 303
```

```suggestion
    weight: 304
```

looks like the previous item in the menu is also 303, so this one should be incremented!

---

oh my gosh.. 🤦 good catch :)